src/com/senlainc/warsaw/tyurin/Test.java

```diff
@@ -0,0 +1,208 @@
+package com.senlainc.warsaw.tyurin;
+
+import com.senlainc.warsaw.tyurin.dao.CraftsmanDAO;
+import com.senlainc.warsaw.tyurin.dao.GaragePlaceDAO;
+import com.senlainc.warsaw.tyurin.dao.OrderDAO;
+import com.senlainc.warsaw.tyurin.entity.Craftsman;
+import com.senlainc.warsaw.tyurin.entity.GaragePlace;
+import com.senlainc.warsaw.tyurin.entity.Order;
+import com.senlainc.warsaw.tyurin.entity.OrderStatus;
+import com.senlainc.warsaw.tyurin.service.CraftsmanService;
+import com.senlainc.warsaw.tyurin.service.GaragePlaceService;
+import com.senlainc.warsaw.tyurin.service.OrderService;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class Test {
+
+    public static void main(String[] args) {
+        GaragePlaceService garagePlaceService = GaragePlaceService.getInstance();
+        CraftsmanService craftsmanService = CraftsmanService.getInstance();
+        OrderService orderService = OrderService.getInstance();
+
+        Craftsman bob = new Craftsman();
```

Comment:
Mechanics, orders and garage places should be created on Service layer, not here (you can count it as Controller layer).
So pass needed variables to service methods and in there create entities and save them to dao